deployment/charts/templates/fuel-core-deploy.yaml

```diff
@@ -72,10 +94,15 @@ spec:
           volumeMounts:
             - name: {{ .Values.app.volume.pvname }}
               mountPath: "{{ .Values.app.volume.mountPath }}"
+            - name: config
+              mountPath: "/mnt/config"
           env:
             - name: HUMAN_LOGGING
               value: {{ .Values.app.human_logging | quote }}
       volumes:
         - name: {{ .Values.app.volume.pvname }}
           persistentVolumeClaim:
             claimName: {{ .Values.app.volume.claimname }}
+        - name: config
```

Comment:
@Voxelot - this looks right but I am not sure, as I never referred configmaps in this fashion this way

I would like to test just to confirm everything makes sense

typically I use

`spec:
  containers:
    - name: env-var-configmap
      image: nginx:1.7.9 
      envFrom:
        - configMapRef:
            name: example-configmap`

the envFrom to reference the configmap, but again your code may work too.

Reply:
envFrom would set the chainspec file as an env var. But since the chainspec could be quite large, I don't think it would fit into an env var. This approach simply mounts the configmap as a file volume on the container which allows it to be a much larger size.